### ISSUE TITLE
Module not found: 'aws-cognito-sdk' node_modules/react-cognito/dist

### ISSUE THREAD

**Issue by kbrown** [isotoma/react-cognito#5]
My compile is balking thusly when installing react-cognito from package.json/npm:

Module not found: 'aws-cognito-sdk' node_modules/react-cognito/dist

**Comment by elsenhuc** [comment]
+1

**Comment by elsenhuc** [comment]
+ docu being horrible!

**Comment by kbrown** [comment]
I had to do some post install hacking.  You can read here about the aws-cognito-sdk: https://github.com/aws/amazon-cognito-identity-js

**Comment by menisweet** [comment]
i have it to! poor docs

**Comment by alex2** [comment]
yeah bad doccies

**Comment by sHtev** [comment]
docs need a doc!

**Comment by ddorsett** [comment]
ok, just hit this. Docs are really horrible,.. kbrown or others: what was the post-install hacking?

**Comment by ddorsett** [comment]
Well, never mind- I decided to just drop down a level and use the cognito identity javascript library directly than mess with these wrappers

**Comment by craigtsmith** [comment]
Guys.... https://github.com/isotoma/react-cognito/tree/master/examples/htdocs/vendor

**Comment by nalbion** [comment]
@kbrown can you share your post-install hack?

**Comment by marcmillien** [comment]
@kbrown what was your trick to make it work? ^^

**Comment by kbrown** [comment]
@marcmillien I had to recreate the functionality of sdkshim.js and test.config.js in my project.   I'm not sure that @Nokrosis' solution isn't better if Amazon isn't going to make the aws-cognito-sdk just work.

**Comment by marcmillien** [comment]
Thanks ^^. You'r right, the @Nokrosis PR is probably the best solution, let's hope it will be merged soon :).

**Comment by zsy2053** [comment]
Hi is this problem solved? I have the same error

**Comment by AlperKarapinar** [comment]
Hi, we did solve this by aliasing aws-cognito-sdk inside webpack config.

```javascript
resolve: {
  ...
    alias: {
     "aws-cognito-sdk": "amazon-cognito-identity-js/dist/aws-cognito-sdk.min.js"
    }
  },
```

**Comment by dibell** [comment]
fixed in 1.5.0
